### ISSUE TITLE
Crash when summon a cactem

### ISSUE THREAD

**Issue by TopsailCBD** [CGessinger/Creatures-And-Beasts-Minecraft-Mod#40]
The game would crash if a cactem spawns when there is no other cactem. 
Details: The first cactem summoned by the spawn egg would cause the game to crash, returning to the game the cactem is still there, then I can spawn another cactems without crash. Kill all the cactems and summon one again, the game would crash again. One day I was flying nearing a dessert biome and the game crashed, it might be the same reason. 
Version: Minecraft 1.19.2, CNB 1.4.0, Forge 43.1.43

Crash Report:
[crash-2022-11-05_19.19.14-server.txt](https://github.com/bonsaistudi0s/Creatures_And_Beasts-Forge/files/9945139/crash-2022-11-05_19.19.14-server.txt)

**Comment by Dariensg** [comment]
Ah, yes! As I suspected, this is the issue that I've attempted to fix in the upcoming release. I've never actually been able to make this crash happen myself, so I don't know for sure that my fix works, but I've made some changes that should hopefully alleviate it. Hopefully, I should be able to release the new version in the next couple days. Just trying to see if I can solve some spawning issues occurring in 1.19.
